openhands/server/routes/public.py

```diff
@@ -65,4 +65,4 @@ async def get_config() -> dict[str, Any]:
     Returns:
         dict[str, Any]: The current server configuration.
     """
-    return server_config.get_config()
+    return dict(server_config.get_config())
```

Comment:
Hmm shouldn't be necessary to add `dict` here, doesn't hurt I guess but `get_config` should already be typed with `dict` - mypy should be able to catch the type compatibility here